WcaOnRails/app/helpers/incidents_helper.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module IncidentsHelper
+  def class_and_text_for_status(incident)
+    if incident.resolved_at
+      ["success", "Resolved"]
+    else
+      ["warning", "Pending"]
+    end
+  end
+
+  def class_and_text_for_digest(incident)
+    if incident.digest_worthy
+      if incident.digest_sent_at
+        ["success", "Sent"]
+      else
+        ["warning", "Pending"]
+      end
+    else
+      ["success", "Not needed"]
```

Comment:
@thewca/wrc-team any suggestion to improve the wording here?
It's basically only displayed to us so maybe it doesn't matter that much, but I wasn't fully satisfied with "Not needed" for the column "Sent in digest" when the incident is not digest-worthy.

Reply:
What about leaving it blank?

Reply:
Why not, it actually highlights better important incidents!